model/countries/AU/AU-parsing-rules.yaml

```diff
@@ -0,0 +1,304 @@
+regex_definitions:
+  kBuildingValueRe:  # Regex for type "building"
+    regex_concat:
+      parts:
+      - regex_fragment: \d+  # House number
+      # Covering cases - housnumber with alphabetic suffix, housnumber range.
+      - regex_fragment: '(?:'
+      - regex_fragment: \s*[[:alpha:]]\b # Optional suffix for building ids with a letter suffix, e.g. 12a.
+      - regex_fragment: '|'
+      - regex_fragment: \s*?[-]\s*\d+ # Housnumber range (10-17)
+      - regex_fragment: ')?'
+      wrap_non_capture: false
+
+  kUnitTypeLiteralRe:  # Regex for "unit-type"
+    regex_fragment: (?:apt\.?|apartment|se\.?|ste\.?|suite|unit|u\.?|room|rm\.?|shop|po box|ms\.?|mail shop|mb\.?|marine berth|vlla\.?|villa|we\.?|warehouse|office|off\.?|fy\.?|factory|sl\.?|stall|site|shed)
+
+  kUnitNameValueRe:  # Regex for "unit-name"
+    regex_fragment: (?:\d+\w?\b|\w\b)
+
+  kFloorLiteralRe:  # Regex for floor literal
+    regex_fragment: (?:level|l\.?|floor|fl\.?|basement|b\.?|ground floor|g\.?|lower ground floor|lg\.?|upper ground floor|ug\.?|mezzanine|m\.?)
```

Comment:
How about wrapping this with `(?:\b` ... `)`? so that not every word ending in "l" or "m" matches?